trace/requirements.txt

```diff
@@ -1,4 +1,6 @@
 google-cloud-trace==0.23.0
 opencensus==0.7.6
 opencensus-ext-stackdriver==0.7.2
+opencensus-ext-flask==0.7.3
```

Comment:
nit: Why is this file being updated? Isn't this for a different sample?

Reply:
Please see comment above. Also I think eventually we are going to move cloud-trace-demo-app to the top folder level and either merge or move this file to a subfolder.